src/app/backend/dashboard.go

```diff
@@ -244,6 +244,13 @@ func handleFatalInitError(err error) {
 		"https://github.com/kubernetes/dashboard/wiki/FAQ", err)
 }
 
+/**
+ * Handles fatal init errors encountered during service cert loading.
+ */
+func handleFatalInitServingCertError(err error) {
+	log.Fatalf("Error while loading dashboard server certificates. Reason: %s", err)
```

Comment:
If this is added then the previous method needs to be changed. Certs are mentioned there and this is no longer a good example.

Reply:
The other message talks about "invalid apiserver certificates". My understanding is that if you were to connect to the api-server using https there could actually be error messages related to certificates created by that. The message seems to be written for that case.

This is pretty much why it was so confusing before. Spent a whole bunch of time trying to figure out what could possibly wrong with the api-server cert I did not touch before realizing the dashboard certificate configuration was at fault.

Reply:
Okay, that makes sense.

/approve

I will let @floreks LGTM it.